app/src/main/AndroidManifest.xml

```diff
@@ -13,9 +13,21 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.MVVMFoundation"
         tools:targetApi="31">
+        <activity
+            android:name=".view.intro.IntroActivity"
+            android:exported="false"
+            android:screenOrientation="portrait"
+            android:theme="@style/Theme.MVVMFoundation.NoActionBar"/>
         <activity
             android:name=".MainActivity"
-            android:exported="true">
+            android:exported="false"
```

Comment:
I set it to false to .MainActivity and true to .SplashActivity. It's supposed to set the splash screen as the first view. I'm wrong?

Reply:
I don't think that is what determines the first view. The attribute `android:exported` determines if the <activity> can be called from outside **this** app. Maybe it forces the android launcher to open that screen because is the only one available outside of **this** app, not sure tho. 

https://developer.android.com/guide/topics/manifest/activity-element